test/foundry/BullaFrendLend/BullaFrendLend.t.sol

```diff
@@ -298,4 +298,321 @@ contract TestBullaFrendLend is Test {
         (uint24 interestBPS, uint40 termLength, uint128 loanAmount, address offerCreditor, ,, ) = bullaFrendLend.loanOffers(loanId);
         assertEq(offerCreditor, address(0), "Offer should be deleted after rejection");
     }
+
+    function testPartialLoanPayments() public {
+        vm.prank(creditor);
+        weth.approve(address(bullaFrendLend), 2 ether);
+        
+        vm.prank(creditor);
+        weth.approve(address(bullaClaim), 2 ether);
+        
+        vm.prank(debtor);
+        weth.approve(address(bullaClaim), 2 ether);
+        
+        vm.prank(debtor);
+        weth.approve(address(bullaFrendLend), 2 ether);
+        
+        LoanOffer memory offer = LoanOffer({
+            interestBPS: 500,
+            termLength: 30 days,
+            loanAmount: 1 ether,
+            creditor: creditor,
+            debtor: debtor,
+            description: "Partial Payments Test Loan",
+            token: address(weth)
+        });
+
+        uint256 initialCreditorWeth = weth.balanceOf(creditor);
+        uint256 initialDebtorWeth = weth.balanceOf(debtor);
+
+        vm.prank(creditor);
+        uint256 loanId = bullaFrendLend.offerLoan{value: FEE}(offer);
+
+        bullaClaim.permitCreateClaim({
+            user: debtor,
+            operator: address(bullaFrendLend),
+            approvalType: CreateClaimApprovalType.Approved,
+            approvalCount: 1,
+            isBindingAllowed: true,
+            signature: sigHelper.signCreateClaimPermit({
+                pk: debtorPK,
+                user: debtor,
+                operator: address(bullaFrendLend),
+                approvalType: CreateClaimApprovalType.Approved,
+                approvalCount: 1,
+                isBindingAllowed: true
+            })
+        });
+
+        vm.prank(debtor);
+        uint256 claimId = bullaFrendLend.acceptLoan(loanId, ClaimMetadata("", ""));
```

Comment:
accept loan shouldn't have any metadata on it, I don't believe.

If there is any metadata, it is when offering the loan.